### PR TITLE
feat: Create the dev sub-command

### PR DESCRIPTION
**Pull request by taorepoara**
#19 

## Checklist
- [x] I didn't over-scope my PR
- [x] My PR title matches the [commit convention](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I did not include breaking changes
- [ ] I included unit tests that cover my changes
- [x] I added/updated the documentation about my changes
- [x] I made my own code-review before requesting one

## Description of the changes
Add the dev command with it basic implementation: build, start and logs(app).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,7 +87,9 @@ OPTIONS:
 
 SUBCOMMANDS:
     build    Build your app in release mode
+    dev      Starts the dev mode
     help     Print this message or the help of the given subcommand(s)
+    init     Generates dockerfile and docker compose file with the init command
```
Comment (jonas-martinez) [inline README.md:92]:
```suggestion
    init     Generates Dockerfile and Docker Compose files
```

File: README.md [hunk 2]
```diff
@@ -103,6 +105,8 @@ This tools contains many subcommands to help you doing what you need.
 - [start](#start): starts your app previously built with the build command
 - [logs](#logs): displays output from the containers
 - [stop](#stop): stops your app previously started with the start command
+- [dev](#dev): starts your app in dev mode
+- [init](#init): Generates Docker and Docker Compose files
```
Comment (jonas-martinez) [inline README.md:109]:
```suggestion
- [init](#init): generates Dockerfile and Docker Compose files
```

File: README.md [hunk 3]
```diff
@@ -212,6 +216,40 @@ OPTIONS:
...
+    lenra dev [OPTIONS]
+
+OPTIONS:
+        --config <CONFIG>    The app configuration file [default: lenra.yml]
+    -h, --help               Print help information
+```
+
+#### init
+
+This subcommand generates Docker and Docker Compose files.
```
Comment (jonas-martinez) [inline README.md:238]:
```suggestion
This subcommand generates Dockerfile and Docker Compose files.
```

File: README.md [hunk 4]
```diff
@@ -212,6 +216,40 @@ OPTIONS:
...
+
+```bash
+$ lenra init --help
+lenra-init 
+Generates dockerfile and docker compose file with the init command
```
Comment (jonas-martinez) [inline README.md:243]:
```suggestion
Generates Dockerfile and Docker Compose files
```

**Review by jonas-martinez — CHANGES_REQUESTED** [review]

**Comment by jonas-martinez** [comment]
Looks good to me, just some rephrasing to do so that the documentation follows one model.

File: README.md (hunk 1 above)
Comment (taorepoara) [reply README.md:92]: I just copied the help result. We need to change the comment in the command enum to change it: https://github.com/lenra-io/lenra_cli/blob/cc4bc8dc886d8811b81a3d90fd52c84658a94636/src/cli/mod.rs#L37

File: README.md (hunk 2 above)
Comment (taorepoara) [reply README.md:109]: It not just the Dockerfile, but also the `.dockerignoer`

File: README.md (hunk 3 above)
Comment (taorepoara) [reply README.md:238]: It not just the `Dockerfile`, but also the `.dockerignore`

File: README.md (hunk 4 above)
Comment (taorepoara) [reply README.md:243]: I just copied the help result. We need to change the comment in the command enum to change it: https://github.com/lenra-io/lenra_cli/blob/cc4bc8dc886d8811b81a3d90fd52c84658a94636/src/cli/mod.rs#L37

**Review by jonas-martinez — CHANGES_REQUESTED** [review]
The CI is failing, could you please try to fix it ?

**Review by jonas-martinez — APPROVED** [review]

**Pull request merged**